sdk/resourcemanager/Azure.ResourceManager/src/Resources/Generated/BasePolicyDefinitionResource.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.ResourceManager.ManagementGroups;
+
+namespace Azure.ResourceManager.Resources
+{
+    /// <summary>
+    /// the base resource of policy definition
+    /// </summary>
+    public abstract class BasePolicyDefinitionResource : ArmResource
```

Comment:
I think the naming convention that we want should match the common Data so in this case it would be `PolicyDefinitionResource`

Reply:
I will apply this naming convention in the actual implementation